{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2020-05-01T01:06:13",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://docs.djangoproject.com/en/dev/releases/security/"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://groups.google.com/forum/#%21topic/django-announce/3oaB2rVH3a0"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/"
                },
                {
                    "name": "USN-4224-1",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_UBUNTU"
                    ],
                    "url": "https://usn.ubuntu.com/4224-1/"
                },
                {
                    "name": "DSA-4598",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_DEBIAN"
                    ],
                    "url": "https://www.debian.org/security/2020/dsa-4598"
                },
                {
                    "name": "20200108 [SECURITY] [DSA 4598-1] python-django security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_BUGTRAQ"
                    ],
                    "url": "https://seclists.org/bugtraq/2020/Jan/9"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://security.netapp.com/advisory/ntap-20200110-0003/"
                },
                {
                    "name": "FEDORA-2020-adb4f0143a",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_FEDORA"
                    ],
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD/"
                },
                {
                    "name": "GLSA-202004-17",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_GENTOO"
                    ],
                    "url": "https://security.gentoo.org/glsa/202004-17"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2019-19844",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)"
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://docs.djangoproject.com/en/dev/releases/security/",
                            "refsource": "MISC",
                            "url": "https://docs.djangoproject.com/en/dev/releases/security/"
                        },
                        {
                            "name": "https://groups.google.com/forum/#!topic/django-announce/3oaB2rVH3a0",
                            "refsource": "MISC",
                            "url": "https://groups.google.com/forum/#!topic/django-announce/3oaB2rVH3a0"
                        },
                        {
                            "name": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/",
                            "refsource": "CONFIRM",
                            "url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/"
                        },
                        {
                            "name": "USN-4224-1",
                            "refsource": "UBUNTU",
                            "url": "https://usn.ubuntu.com/4224-1/"
                        },
                        {
                            "name": "DSA-4598",
                            "refsource": "DEBIAN",
                            "url": "https://www.debian.org/security/2020/dsa-4598"
                        },
                        {
                            "name": "20200108 [SECURITY] [DSA 4598-1] python-django security update",
                            "refsource": "BUGTRAQ",
                            "url": "https://seclists.org/bugtraq/2020/Jan/9"
                        },
                        {
                            "name": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html",
                            "refsource": "MISC",
                            "url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html"
                        },
                        {
                            "name": "https://security.netapp.com/advisory/ntap-20200110-0003/",
                            "refsource": "CONFIRM",
                            "url": "https://security.netapp.com/advisory/ntap-20200110-0003/"
                        },
                        {
                            "name": "FEDORA-2020-adb4f0143a",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD/"
                        },
                        {
                            "name": "GLSA-202004-17",
                            "refsource": "GENTOO",
                            "url": "https://security.gentoo.org/glsa/202004-17"
                        }
                    ]
                }
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-07-02T13:03:04.405Z"
                },
                "references": [
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://docs.djangoproject.com/en/dev/releases/security//DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://groups.google.com/forum/#%21topic/django-announce/3oaB2rVH3a0/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases//DEMO_2024_06_19"
                    },
                    {
                        "name": "USN-4224-1",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_UBUNTU",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://usn.ubuntu.com/4224-1//DEMO_2024_06_19"
                    },
                    {
                        "name": "DSA-4598",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_DEBIAN",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.debian.org/security/2020/dsa-4598/DEMO_2024_06_19"
                    },
                    {
                        "name": "20200108 [SECURITY] [DSA 4598-1] python-django security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_BUGTRAQ",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://seclists.org/bugtraq/2020/Jan/9/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://security.netapp.com/advisory/ntap-20200110-0003//DEMO_2024_06_19"
                    },
                    {
                        "name": "FEDORA-2020-adb4f0143a",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_FEDORA",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD//DEMO_2024_06_19"
                    },
                    {
                        "name": "GLSA-202004-17",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_GENTOO",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://security.gentoo.org/glsa/202004-17/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_snapshot"
                        ],
                        "url": "https://docs.djangoproject.com/en/dev/releases/security//adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_snapshot"
                        ],
                        "url": "https://groups.google.com/forum/#%21topic/django-announce/3oaB2rVH3a0/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_snapshot"
                        ],
                        "url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases//adp-test-6-28"
                    },
                    {
                        "name": "USN-4224-1",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_UBUNTU",
                            "x_snapshot"
                        ],
                        "url": "https://usn.ubuntu.com/4224-1//adp-test-6-28"
                    },
                    {
                        "name": "DSA-4598",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_DEBIAN",
                            "x_snapshot"
                        ],
                        "url": "https://www.debian.org/security/2020/dsa-4598/adp-test-6-28"
                    },
                    {
                        "name": "20200108 [SECURITY] [DSA 4598-1] python-django security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_BUGTRAQ",
                            "x_snapshot"
                        ],
                        "url": "https://seclists.org/bugtraq/2020/Jan/9/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_snapshot"
                        ],
                        "url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_snapshot"
                        ],
                        "url": "https://security.netapp.com/advisory/ntap-20200110-0003//adp-test-6-28"
                    },
                    {
                        "name": "FEDORA-2020-adb4f0143a",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_FEDORA",
                            "x_snapshot"
                        ],
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD//adp-test-6-28"
                    },
                    {
                        "name": "GLSA-202004-17",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_GENTOO",
                            "x_snapshot"
                        ],
                        "url": "https://security.gentoo.org/glsa/202004-17/adp-test-6-28"
                    }
                ]
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2019-19844",
        "datePublished": "2019-12-18T18:07:11",
        "dateReserved": "2019-12-17T00:00:00",
        "dateUpdated": "2024-07-02T13:03:04.405Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}